### ISSUE TITLE
Support : on classe names

### ISSUE THREAD

**Issue by DahmaniAdame** [builderius/builderius#8]
Can we somehow make classes support `:` on them? 

I like to use https://css.master.co/ and it will be very convenient if I can use classes like `bg:gray` for example.

The library seems to be working. All I need is a way to add `:` on classes.

**Comment by den-v** [comment]
@DahmaniAdame I checked that library and allowing ":" in class names will not make possible to use full functionality of css.master. We will allow all symbols in class name validator except space and double quotes.

**Comment by DahmaniAdame** [comment]
> I checked that library and allowing ":" in class names will not make possible to use full functionality of css.master.

@den-v yes, you are right. Other symbols will also need to be allowed to get the most out of the framework. noticing that. 

> We will allow all symbols in class name validator except space and double quotes.

Sounds good! That would benefit other CSS frameworks like Tailwind. 

Thanks in advance 🙏

**Comment by seaj1one** [comment]
Would we also be able to use things like 

Double hyphen or double underscore, too, so we can use BEM. 

Here's an example

https://css-tricks.com/bem-101/
